pkg/azure/resources.go

```diff
@@ -33,17 +33,8 @@ import (
 )
 
 const (
-	virtualMachineTypeName        = "Microsoft.Compute/virtualMachines"
-	managedClusterTypeName        = "Microsoft.ContainerService/managedClusters"
-	localNetworkGatewayTypeName   = "Microsoft.Network/localNetworkGateways"
-	diskTypeName                  = "Microsoft.Compute/disks"
-	connectionTypeName            = "Microsoft.Network/connections"
-	networkInterfaceTypeName      = "Microsoft.Network/networkInterfaces"
-	networkSecurityGroupTypeName  = "Microsoft.Network/networkSecurityGroups"
-	publicIPAddressTypeName       = "Microsoft.Network/publicIPAddresses"
-	virtualNetworkGatewayTypeName = "Microsoft.Network/virtualNetworkGateways"
-	virtualNetworkTypeName        = "Microsoft.Network/virtualNetworks"
-	networkWatcherTypeName        = "Microsoft.Network/networkWatchers"
+	virtualMachineTypeName = "Microsoft.Compute/virtualMachines"
```

Comment:
Not to require more iterations than needed, but I wouldn't be opposed to having these in the utils too.